modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -570,14 +571,23 @@ describe('TSS Ecdsa Utils:', async function () {
         userKeyShare.nShares[3], backupKeyShare.nShares[3],
       ]);
 
+      const userChallenge = await rangeProof.generateNTilde(3072);
       const bitgoChallenge = await rangeProof.generateNTilde(3072);
```

Comment:
```suggestion
      const [ userChallenge,bitgoChallenge]  = await Promise.all([rangeProof.generateNTilde(3072). await rangeProof.generateNTilde(3072)];
```

could speed up tests a bit